zombienet/README.md

```diff
@@ -0,0 +1,23 @@
+**Zombienet for macos**
+
+Download from [zombienet](https://github.com/paritytech/zombienet/releases)
+
+or
+
+`wget https://github.com/paritytech/zombienet/releases/download/v1.3.30/zombienet-macos `
+
+`chmod +x zombienet-macos`
+
+`./zombienet-macos --help`
+
+Build Polkadot
+
+`cargo build --release`
+
+Build Foucoco
+
+`cargo build --release`
```

Comment:
And here we could be more specific and say which package needs to be built or which packages can be built. ie. say something like 
```
cargo build --release --package {amplitude/pendulum/foucoco}-runtime
```

Reply:
Oh. you right we should be more specific. 
We should build pendulum node. and not runtime
and the foucoco/pendulum or amplitude you are able to peek specifying corresponding value in zombienet.toml file.
will change the readme.
@ebma thanks.